README.md

```diff
@@ -1,290 +1,2 @@
-# A Byte of Python
```

Comment:
Is this change required? This Markdown file is what people who visit https://github.com/swaroopch/byte-of-python currently see.

Reply:
You're totally right. I was newbie and didn't find out that change. I'll review changes and send the request again.